gammapy/estimators/map/ts.py

```diff
@@ -745,6 +849,52 @@ def estimate_errn_errp(self, dataset, result):
         )
         return {"norm_errn": flux_errn, "norm_errp": flux_errp}
 
+    def estimate_scan_local(self, dataset, result):
+        """Compute likelihood profile around best norm value.
+
+        Parameters
+        ----------
+        dataset : `SimpleMapDataset`
```

Comment:
This class is not exposed in the documentation. How can a user know what is this input?

Reply:
It is used only within the TSMapEstimator, users don't have to use it.